### PR TITLE
Upgrade out-of-date dependencies

### PR DESCRIPTION
**Pull request by efischer19**
## [TNL-4841](https://openedx.atlassian.net/browse/TNL-4841)

"Upgrading" a few out-of-date dependencies to make Gemnasium happy.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ git+https://github.com/edx/edx-submissions.git@1.1.0#egg=edx-submissions==1.1.0
 boto>=2.32.1,<3.0.0
 celery==3.1.18
 defusedxml==0.4.1
-django>=1.8,<1.9
+django==1.8.13
```
Comment (efischer19) [inline requirements/base.txt]: This is the version that's already installed via edx-platform, specifying it exactly vs a range should resolve the warning.

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "jasmine": "^2.3.0",
     "phantomjs": "^1.9.11",
     "sinon": "^1.10.3",
-    "uglify-js": "2.3.6",
+    "uglify-js": "2.6.3",
```
Comment (efischer19) [inline package.json]: @bjacobel or @andy-armstrong - are there any large concerns with this upgrade, or additional testing I can do? This is used in the Makefile for `make javascript`, I already ran that and the minified js had no differences at all.

**Comment by efischer19** [comment]
Local python and javascript tests all pass without issue, I'm going to get a sandbox spun up for acceptance tests before merging.

File: package.json (hunk 2 above)
Comment (bjacobel) [inline package.json]: If there's no diff before and after the upgrade I'd say it should be good to go.

Comment (efischer19) [inline package.json]: Great, thanks @bjacobel!

Comment (andy-armstrong) [inline package.json]: I agree. Can you change the rule to be ``"^2.6.3"`` too for consistency.

Comment (efischer19) [inline package.json]: Oh cool, I didn't even know that was a thing until now, thanks! Adding.

File: requirements/base.txt [hunk 3]
```diff
@@ -6,13 +6,13 @@
 git+https://github.com/edx/XBlock.git@xblock-0.4.10#egg=XBlock==0.4.10
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@1.1.0#egg=edx-submissions==1.1.0
+git+https://github.com/edx/edx-submissions.git@efischer/gemnasium#egg=edx-submissions==1.1.0
```
Comment (efischer19) [inline requirements/base.txt]: `edx-submissions` has the exact same alert and can be fixed exactly the same way, so I'm going to bundle the changes together.

Comment (efischer19) [inline requirements/base.txt]: Note that I have the branch specified for testing purposes, it'll be a proper version bump once merged.

See https://github.com/edx/edx-submissions/pull/46

**Comment by efischer19** [comment]
@cahrens @jcdyer Would you mind giving this a quick review? I've verified that all tests continue to pass with the version bumps, as seen at http://jenkins.edx.org:8080/view/ora2/job/ora2-acceptance-tests/918/.

Once thumbsed, I'll
- merge update to edx-submissions, and make a new release there
- update this branch with that requirement, and
- merge to master

**Comment by cahrens** [comment]
👍

**Comment by jcdyer** [comment]
:+1: pending updating the requirements to edx-submissions == 1.1.1 (Don't forget to get both the `@` and the `egg=`

**Pull request merged**